tests/simple.rs

```diff
@@ -24,6 +24,17 @@ fn simple() -> Result<(), failure::Error> {
     Ok(())
 }
 
+#[test]
+fn actions_on_tab_wont_hang_after_browser_drops() -> Result<(), failure::Error> {
+    logging::enable_logging();
+    let tab = {
```

Comment:
I can't tell if the fix is actually correct or not, I don't know the underlying code well enough. This is the only nitpick, which could be written as

```rust
// Issue 32, dropping Browser while Tab was still in use caused a hang
let (_, browser, tab) = dumb_server(....);
drop(browser);
assert!(tab.find_element("div#foobar").is_err());
```
The explicit `drop` makes it more obvious what the test is about; the drop being implicit in the `let tab = { ... }`-variant.

Reply:
Thanks, fixed. I initially tried `browser.drop()` and `Drop::drop(browser)` and they didn't work. I didn't realise there was also `drop`!